Makefile

```
@@ -86,12 +86,21 @@ $(AMD64_BINS): GOARCH=amd64
 $(ARM64_BINS): GOARCH=arm64
 
 .PHONY: build
-build: $(DRIVER_BINS)
+build: vet fmt $(DRIVER_BINS)
 
-$(DRIVER_BINS): vet fmt
+$(DRIVER_BINS):
 	@CGO_ENABLED=0 GOTOOLCHAIN=local GOOS=linux GOARCH=$(GOARCH) \
 		go build -ldflags "${LDFLAGS}" -o $@ ./cmd
 
+# For CI use. Runs `make build` inside Docker
+# container using an official golang image.
+.PHONY: docker-build
+docker-build:
+	docker run --rm \
+		--workdir /go/src/github.com/racktopsystems/brickstor-csi-driver \
+		--volume ./:/go/src/github.com/racktopsystems/brickstor-csi-driver \
+		${BUILD_IMAGE} make -j4 build VERSION=${VERSION}
```

I think this `-j` bit should be dynamic. If we target the mac and linux as our build platforms, we can use something like `nproc`. In my suggestion I am also replacing braces with parentheses.
```suggestion
		$(BUILD_IMAGE) make -j$(nproc) build VERSION=$(VERSION)
```